docs/install.md

```diff
@@ -0,0 +1,77 @@
+# Install
+
+This workflow will install the two operators, [Backend] and [Cluster-Agent], by pulling their container images from the container registry respectively.
+Thus, to deploy them into Kubernetes, make sure you have _already build and push_ the container image to your registry.
+For example: `make docker-build docker-push IMG=quay.io/pgeorgia/gitops-service:latest`
+
+As soon as you have a container image into your registry, you are ready to deploy the backend controller:
+Given your `kube-config` is already pointing to your Kubernetes cluster, then do:
+
+```bash
+make install-all-k8s IMG=quay.io/pgeorgia/gitops-service:latest # replace the IMG var with yours
+```
+
+This will automatically install all the required components into your Kubernetes cluster, in the `gitops` namespace.
+Notice that, the [Cluster-Agent] operator requires the `argocd` namespace which will also be created as well.
```

Comment:
Are we deploying upstream Argo CD ?

Reply:
It will use any Argo CD distribution you can see an example of us using OpenShift GitOps in the M2 demo:
https://github.com/redhat-appstudio/managed-gitops/blob/3f5de2ebfd28b0b7fa20d4ac192b5af98a246caf/examples/m2-demo/setup-on-openshift.sh#L49